sec_certs/sample/common_criteria.py

```diff
@@ -189,7 +198,7 @@ class CCHeuristics(ComplexSerializableType):
         verified_cpe_matches: Optional[Set[str]] = field(default=None)
         related_cves: Optional[Set[str]] = field(default=None)
         cert_lab: Optional[List[str]] = field(default=None)
-        cert_id: Optional[str] = field(default=None)
+        cert_id: Optional[str] = field(default=None)  # TODO - delete this
```

Comment:
Please, see: https://github.com/crocs-muni/sec-certs/pull/166#discussion_r801129966
Sorry for the confusion :(